src/Pie.js

```diff
@@ -100,10 +156,15 @@ Pie.propTypes = {
   innerRadius: PropTypes.number,
   backgroundColor: PropTypes.string,
   strokeCap: PropTypes.string,
-}
+  dividerProps: PropTypes.exact({
+    size: PropTypes.number.isRequired,
+    color: PropTypes.string.isRequired,
```

Comment:
I'm wondering if we could simplify the api to use the `backgroundColor` as the divider's color, that approach seems intuitive, if you need other colors, you could just inset those dividers by yourself like `[20, 30] -> [19, 2, 29]`
The reason is that we should avoid introducing object prop for shallow comparison
In that case we could only introduce `dividerSize`, another concern is that `divider size` is a bit confusing as we are actually referring to percentage here

Reply:
Also if we use the backgroundColor as the divider's color, we can simply implement the divider via `ArcShape` so reduce half of the size on both sides, in that way we don't need `Dividers` anymore

Reply:
@nihgwu We can do that. The complication comes with `round` strokeCap. In order to achieve the UI above I had to paint over the white to create a divider. Can you think of a different way to accomplish that?

Reply:
Also I found the square `strokeCap` is totally broken and we should probably disallow it

Reply:
I'm wondering if we could only support dividers for `butt` strokeCap, but I think we could still add the overlay in `ArcShape` as well, for `butt` we just leave the gap empty, and for `round` we render extra arc

Reply:
I could implement this. We leave gaps for `butt` and use the `backgroundColor` when rendering an overlay for `round`. Does that sound good?

Reply:
I made the above changes, let me know what you think